app/views/about_users/_form.html.haml

```diff
@@ -0,0 +1,16 @@
+%p
```

Comment:
Can you use the bootstrap classes such as `.form-group` etc? Refer to other forms in the codebase.

Reply:
👍